src/components/pages/home/elements/WeatherBlock.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class WeatherBlock extends Component {
+  render() {
+    return (
+      <div className="forecast_block">
```

Comment:
And again - `forecast-block` (although the component is called `WeatherBlock` and only fetches existing data so this should probably be `weather-block`)